aws/logs_monitoring/parsing.py

```diff
@@ -527,6 +528,47 @@ def cwevent_handler(event, metadata):
     yield data
 
 
+# Extract findings from Security Hub logs
+# and move them to the root of the log
+def separate_security_hub_findings(event):
+    if (
+        event.get(DD_SOURCE) == "securityhub"
+        and event.get("detail", {}).get("findings") is not None
```

Comment:
`is not None` seems redundant?

Reply:
It is in most cases. This is just to cover the case of customers sending a non-standard Security Hub log (one with no findings). In that case, we would be doing extra work to remove the original log in the events array only to replace it with itself. I guess it's a question of whether that savings is worth a cumbersome if statement.

Reply:
Ah, sorry, I guess you mean to rewrite it simply as:

```
and event.get("detail", {}).get("findings")
```

I agree that's better. 👍